srv/modules/runners/remove.py

```diff
@@ -36,6 +36,13 @@ def osd(*args, **kwargs):
 
     local = salt.client.LocalClient()
 
+    # Specifically for Salt integration testing
+    # The 'arg' keyword is passed as a kwarg to the salt.runner
+    # See delay.sls and multiple.sls for examples
+    if not args:
```

Comment:
In a non-testing environment, wouldn't this mean

`salt-run remove.osd`  # <- without passing any OSDs?

I'd assume this is here we should drop a help message

Reply:
Addressed... had to move the Salt check up as well.